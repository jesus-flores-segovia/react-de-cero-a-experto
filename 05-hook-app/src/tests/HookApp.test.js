import "@testing-library/jest-dom";
import { shallow } from "enzyme";
import {HookApp} from "../HookApp";

describe("Tests inside file HookApp.test.js", () => {

    test("'HookApp' component must show correctly", () => {
        const wrapper = shallow(<HookApp></HookApp>)
        expect(wrapper).toMatchSnapshot();
    });

});