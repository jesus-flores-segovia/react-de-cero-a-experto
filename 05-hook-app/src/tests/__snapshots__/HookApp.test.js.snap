// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests inside file HookApp.test.js 'HookApp' component must show correctly 1`] = `
<div>
  <h1>
    HookApp
  </h1>
</div>
`;
